scriptmodules/emulators/fs-uae.sh

```diff
@@ -47,16 +49,18 @@ function remove_fs-uae() {
 function configure_fs-uae() {
     mkRomDir "amiga"
 
-    mkUserDir "$home/.config"
+    # copy configuring start script
+    mkdir "$md_inst/bin/"
+    cp "$scriptdir/scriptmodules/$md_type/$md_id/fs-uae.sh" "$md_inst/bin/"
+    chmod +x "$md_inst/bin/fs-uae.sh"
+
     mkUserDir "$md_conf_root/amiga"
-    moveConfigDir "$home/.config/fs-uae" "$md_conf_root/amiga/fs-uae"
+    mkUserDir "$home/Documents/FS-UAE"
+    mkUserDir "$home/Documents/FS-UAE/Configurations"
+    moveConfigDir "$home/Documents/FS-UAE/Configurations" "$md_conf_root/amiga"
 
-    cat > "$romdir/amiga/+Start FS-UAE.sh" << _EOF_
-#!/bin/bash
-fs-uae-launcher
-_EOF_
-    chmod a+x "$romdir/amiga/+Start FS-UAE.sh"
-    chown $user:$user "$romdir/amiga/+Start FS-UAE.sh"
+    # copy default config file
+    cp -v "$scriptdir/scriptmodules/$md_type/$md_id/Default.fs-uae" "$md_conf_root/amiga"
```

Comment:
tiny config - can use iniSet - also make sure we don't overwrite existing configs on upgrade. (configure should not overwrite existing configs).